pkg/datarepair/checker/checker.go

```diff
@@ -30,31 +30,45 @@ var (
 
 // Config contains configurable values for checker
 type Config struct {
-	Interval time.Duration `help:"how frequently checker should audit segments" default:"30s"`
+	MaxRepair        int           `help:"maximum segments that can be repaired concurrently" releaseDefault:"5" devDefault:"1"`
+	Interval         time.Duration `help:"how frequently checker should audit segments" releaseDefault:"30s" devDefault:"0h0m10s"`
+	IrrepairInterval time.Duration `help:"how frequently irrepair checker should audit segments" releaseDefault:"15s" devDefault:"0h0m5s"`
```

Comment:
super nit, but I think I'd prefer to name this "IrreparableInterval" or "IrrepInterval"

Reply:
changed to IrrdbInterval. Just to keep the name in consistent with short form nameing. Let me know